modules/RsPaymentTerminal/manifest.yaml

```diff
@@ -40,6 +40,18 @@ config:
   password:
     description: Password for the Feig terminal.
     type: integer
+  accept_credit_cards_for_all_connectors:
+    description: Indicates wheter credit card should be accepted for all the connectors
+    type: boolean
+    default: true
+  credit_card_connectors:
+    description: >-
+      If the `accept_credit_cards_for_all_connectors` is set to false credit cards
+      will only provide authentication to the given connectors.
+      If list is empty, authentication won't happen.
+    type: string
+    default: "1,2"
```

Comment:
i thought we're doing it the reverse way - default should be empty. in this case we don't specify the connectors at all